.gitignore

```diff
@@ -0,0 +1,29 @@
+*.py[co]
+
+# Packages
+*.egg
+*.egg-info
+dist
+build
+eggs
+parts
+bin
+var
+sdist
+develop-eggs
+.installed.cfg
+
+# Installer logs
+pip-log.txt
+
+#shared objects
+*.so
+
+#config
+cf_units/etc/site.cfg
```

Comment:
I'd like to test the doc building ... so please can we fix this in a follow-up PR.